etl/steps/data/garden/health/2024-04-22/gpei_funding.meta.yml

```diff
@@ -0,0 +1,53 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Polio
+        - Eradication of Diseases
+    unit: "current US$"
```

Comment:
Is there any place where current is stated? Because it can be constant as well. But if it's current we can't create line charts, for example, because amounts are not comparable across time. We would needto adjust by inflation